### ISSUE TITLE
Expand animation

### ISSUE THREAD

**Issue by sergeyfitis** [h6ah4i/android-advancedrecyclerview#61]
Hi. Is it possible to change expand group animation?

**Comment by h6ah4i** [comment]
Yes. You can achieve that by impleumenting your own `RecyclerView.ItemAnimator` class. (However, the concept of *item grouping* is not exist at the ItemAnimator's level.)
